canvas_modules/harness/src/client/App.js

```diff
@@ -1983,6 +1991,7 @@ class App extends React.Component {
 				callbacks={callbacks}
 				customControls={[CustomToggleControl, CustomTableControl, CustomEmmeansDroplist]}
 				customConditionOps={[CustomOpMax, CustomNonEmptyListLessThan, CustomOpSyntaxCheck]}
+				light={this.state.light}
```

Comment:
Going off my comment above if you removed this what shows up in the UI?

Reply:
You are right. 
`this.light = true;` in properties-controller was not setting the default value correctly. If I removed this code in App.js, `light` was set to `undefined`. 
Adding `light: true` as default prop in common-properties fixed the issue.